### PR TITLE
doc(self-managed): improve multiregion self-managed readability

### PR DESCRIPTION
**Pull request by leiicamundi**
## Description

This PR aims to improve readability of the self-managed multi-region documentation by adding tables and refactoring some parts. 

## When should this change go live?

It should not be merged until https://github.com/camunda/camunda-docs/pull/4205 is merged.

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@leiicamundi Let me know when this PR is ready for a TW review 👍

**Comment by Langleu** [comment]
@leiicamundi could you rebase the branch, so your change is isolated to what you wanted to restructure.

**Comment by mesellings** [comment]
Thanks @leiicamundi I'll do this for you now 👍

**Review by mesellings — APPROVED** [review]
@leiicamundi I have reviewed and made commits directly to the branch for readability, writing standards (for example, we do not use "e.g.", but rather "for example,", and general grammar.

Great work - happy to approve 👍

File: docs/self-managed/concepts/multi-region/dual-region.md [hunk 1]
```diff
@@ -86,67 +86,73 @@ In the event of a total active region loss, the following data will be lost:
...
-- Managing DNS and incoming traffic
+## Requirements and Limitations
+
+### Minimum Versions
+
+| **Component**       | **Elasticsearch** | **Operate** | **Tasklist** | **Zeebe** | **Zeebe Gateway** | **Camunda Helm Chart**                                   |
+| ------------------- | ----------------- | ----------- | ------------ | --------- | ----------------- | -------------------------------------------------------- |
+| **Minimum Version** | 8.9+\*            | 8.5+        | 8.5+         | 8.5+      | 8.5+              | [9.3+](https://github.com/camunda/camunda-platform-helm) |
+
+\* OpenSearch (both managed and self-managed) is not supported for a dual-region setup.
```
Comment (Langleu) [inline docs/self-managed/concepts/multi-region/dual-region.md:97]: Wasn't a big fan of this in the past already and wondering whether we should just remove it altogether.
We're doing versioned docs, meaning the 8.5 stuff isn't really relevant for 8.6.
We have an extra document on supported environments stating e.g. Elasticsearch version.

Anything 8.5 and 8.6 are compatible. 9.3 is even 8.4 which may even be more confusing to customers.

OpenSearch would still need to be mentioned somewhere.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 2]
```diff
@@ -68,17 +68,16 @@ Running dual-region setups requires the users to be able to detect any regional
...
-
-In case the region is only lost temporarily (for example, due to network hiccups), Zeebe can survive a region loss but will stop processing due to the loss in quorum and ultimately fill up the persistent disk before running out of volume, resulting in the loss of data.
-
-The **failover** phase of the procedure temporarily restores Camunda 8 functionality by removing the lost brokers and the export to the unreachable Elasticsearch instance.
-
-The **failback** phase of the procedure results in completely restoring the failed region to its full functionality. It requires you to have the lost region ready again for the redeployment of Camunda 8.
+1. **Traffic Rerouting:** Reroute traffic to the surviving active region using DNS. (Details on how to manage DNS rerouting depend on your specific DNS setup and are not covered in this guide.)
+2. **Prevent Reconnection:** Ensure that the lost region cannot reconnect before starting the restoration procedure. Reconnection could interfere with a successful recovery during failover and failback.
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:76]: you can cut this point 2. out, it's not applicable for the new procedure anymore.

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 1 above)
Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md:97]: we don't rely on anything DB specific and just rely on the version stated in the supported environments.
With the only exception of OpenSearch and that mostly due to tests missings.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 3]
```diff
@@ -139,15 +138,19 @@ desired={<Five viewBox="140 40 680 500" />}
...
-You have previously ensured that the lost region cannot reconnect during the failover procedure.
+One of the regions is lost, meaning Zeebe:
 
-Due to the Zeebe data replication, no data has been lost.
+- Is unable to process new requests due to losing the quorum
+- Stops exporting new data to Elasticsearch in the lost region
+- Stops exporting new data to Elasticsearch in the survived region
 
-#### Desired state
+You have previously ensured that the lost region cannot reconnect during the failover procedure.
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:151]: same as above not applicable anymore.
We could rephrase it to something along the lines of
```
You have ensured that you fully lost a region and want to start the temporary recovery.
```

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 4]
```diff
@@ -139,15 +138,19 @@ desired={<Five viewBox="140 40 680 500" />}
...
 
-You have removed the lost brokers from the Zeebe cluster. This will allow us to continue processing after the next step and ensure that the new brokers in the failback procedure will only join the cluster with our intervention.
+Due to the Zeebe data replication, no data has been lost.
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:153]: I think copy / pasta as you're repeating yourself with the above current state.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md (hunk 3 above)
Comment (Langleu) [reply docs/self-managed/operational-guides/multi-region/dual-region-ops.md:151]: actually copy / pasta as it's covered in the table above.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 5]
```diff
@@ -342,7 +339,7 @@ curl -XGET 'http://localhost:9600/actuator/exporters'
   </summary>
 </details>
 
-2. Based on the Exporter API you will send a request to the Zeebe Gateway to disable the Elasticsearch exporter to the lost region.
+2. Based on the [Exporter APIs](../../zeebe-deployment/operations/cluster-scaling.md) you will send a request to the Zeebe Gateway to disable the Elasticsearch exporter to the lost region.
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:342]: I thought so first as well but it's not documented there, so removed it for now to not cause confusion.
Afaik not 100% documented yet.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 6]
```diff
@@ -408,13 +405,10 @@ desired={<Eight viewBox="140 40 680 500" />}
...
+| **Details**              | **Current State**                                                                                                   | **Desired State**                                                                                                 |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| **Camunda 8 Setup**      | A standalone region with a fully functional Camunda 8 setup, including Zeebe, Operate, Tasklist, and Elasticsearch. | Restore dual-region functionality by deploying Camunda 8 (Zeebe and Elasticsearch) to the newly restored region.  |
+| **Operate and Tasklist** | Operate and Tasklist are operational in the standalone region.                                                      | Operate and Tasklist should remain disabled to avoid interference during the database backup and restore process. |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:411]: the rework on the wording is not ideal
> Operate and Tasklist should remain disabled to avoid interference during the database backup and restore process.

I'd stay with
> ... need to stay disabled to prevent ...

should has an optional flair to it and makes it sounds like it's okay if you have it enabled and avoid is also not ideal vs prevent. We're not avoiding this, we're trying to prevent it from happening in the first place.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 7]
```diff
@@ -520,21 +514,11 @@ desired={<Nine viewBox="140 40 680 500" />}
...
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Zeebe Cluster Setup**  | Functioning Zeebe cluster within a single region: <br/> Working Camunda 8 installation in the surviving region <br/> Non-participating Camunda 8 installation in the recreated region | Preparing the newly created region to take over and restore the dual-region setup.                                                                              |
+| **Elasticsearch Export** | Currently exporting data to Elasticsearch from the surviving region.                                                                                                                  | Stop Zeebe exporters to prevent new data from being exported to Elasticsearch, allowing for the creation of an Elasticsearch backup.                            |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:520]: Wouldn't that be `Export Capability`?

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 8]
```diff
@@ -520,21 +514,11 @@ desired={<Nine viewBox="140 40 680 500" />}
...
+| **Details**              | **Current State**                                                                                                                                                                     | **Desired State**                                                                                                                                               |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Zeebe Cluster Setup**  | Functioning Zeebe cluster within a single region: <br/> Working Camunda 8 installation in the surviving region <br/> Non-participating Camunda 8 installation in the recreated region | Preparing the newly created region to take over and restore the dual-region setup.                                                                              |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:519]: Wouldn't that be `Camunda 8 Setup`?

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 9]
```diff
@@ -586,13 +570,10 @@ desired={<Ten viewBox="140 40 680 500" />}
...
-
-#### Desired state
-
-You are creating a backup of the main Elasticsearch instance in the surviving region and restore it in the recreated region. This Elasticsearch backup contains all the data and may take some time to be finished.
+| **Details**              | **Current State**                                                                                                        | **Desired State**                                                                                                                                                |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Camunda Components**   | Not reachable by end-users and not processing any new process instances. This state allows for data backup without loss. | Creating a backup of the main Elasticsearch instance in the surviving region and restoring it in the recreated region. Backup process may take time to complete. |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:575]: I'm not sure the desired state brings much value since it's the same as below for the Elasticsearch Backup.

The bit about
> Backup process may take time to complete.
could be added to the desired state below.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 10]
```diff
@@ -878,15 +854,11 @@ desired={<Twelve viewBox="140 40 680 500" />}
...
-Camunda 8 is reachable to the end-user but not yet exporting any data.
-
-#### Desired state
-
-You are initializing a new exporter to the recreated region. This will ensure that both Elasticsearch instances are populated, resulting in data redundancy.
-
-Separating this step from resuming the exporters is essential as the initialization is an asynchronous procedure, and you must ensure it's finished before resuming the exporters.
+| **Details**                 | **Current State**                                   | **Desired State**                                                                                     |
+| --------------------------- | --------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| **Camunda 8 Accessibility** | Reachable to end-users, but not exporting any data. | Start a new exporter to the recreated region.                                                         |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:859]: wouldn't you just call it `Camunda 8 Setup` or `Camunda Components`?

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 11]
```diff
@@ -878,15 +854,11 @@ desired={<Twelve viewBox="140 40 680 500" />}
...
+| **Data Exporting**          | No data export currently in progress.               | Ensure that both Elasticsearch instances are populated for data redundancy.                           |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:860]: I'd remove this step, it's just repeating what has been said already.
Add `Ensure that both Elasticsearch instances are populated for data redundancy. ` to the sentence above.

On it's own doesn't make sense. Who is ensuring that? The start of a new exporter.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 12]
```diff
@@ -878,15 +854,11 @@ desired={<Twelve viewBox="140 40 680 500" />}
...
+| **Procedure Step**          | Data export is paused.                              | Separate the initialization step (asynchronous) and confirm completion before resuming the exporters. |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:861]: It has the same issue as the row above, on it's own it doesn't make much sense as context is missing.

Maybe just remove it.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 13]
```diff
@@ -954,15 +926,10 @@ desired={<Thirteen viewBox="140 40 680 500" />}
...
-Camunda 8 is reachable to the end-user but not yet exporting any data.
-
-Elasticsearch exporters are enabled for both regions, and it's ensured that the operation has finished.
-
-#### Desired state
-
-You are reactivating the existing exporters. This will allow Zeebe to export data to Elasticsearch again.
+| **Details**                 | **Current State**                                                       | **Desired State**                                  |
+| --------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------- |
+| **Camunda 8 Accessibility** | Reachable to end-users, but currently not exporting any data.           | Reactivate existing exporters.                     |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:931]: I think for the details you should align on same wording and just use `Camunda 8 Setup` or just go down to `Camunda 8`

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 14]
```diff
@@ -954,15 +926,10 @@ desired={<Thirteen viewBox="140 40 680 500" />}
...
+| **Elasticsearch Exporters** | Enabled for both regions, with the operation confirmed to be completed. | Allow Zeebe to export data to Elasticsearch again. |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:932]: I'm not sure why you're repeating yourself here.
it's about Camunda 8 functionality and would combine it to one.
Earlier we had points about Elasticsearch in Elasticsearch, which okay made sense. But now we're tackling about Elasticsearch but from a Camunda 8 functionality.

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 15]
```diff
@@ -992,13 +959,10 @@ desired={<Fourteen viewBox="140 40 680 500" />}
...
-#### Current state
-
-Camunda 8 is running in two regions but not yet utilizing all Zeebe brokers. You have redeployed Operate and Tasklist and enabled the Elasticsearch exporters again. This will allow users to interact with Camunda 8 again.
-
-#### Desired state
-
-You have a functioning Camunda 8 setup in two regions and utilizing both regions. This will fully recover the dual-region benefits.
+| **Details**              | **Current State**                                                                                                                  | **Desired State**                                                                             |
+| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| **Camunda 8 Deployment** | Running in two regions, but not yet utilizing all Zeebe brokers. Operate and Tasklist redeployed, Elasticsearch exporters enabled. | Fully functional Camunda 8 setup utilizing both regions, recovering all dual-region benefits. |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:964]: Details name ... would stick to one naming.

**Review by Langleu — CHANGES_REQUESTED** [review]
Overall, cool suggestion.
Not the biggest fan of the detail column as it doesn't align well on the wording. Also some rows are just repeating the same information or completely out of context.

File: docs/self-managed/concepts/multi-region/dual-region.md [hunk 16]
```diff
@@ -86,67 +86,66 @@ In the event of a total active region loss, the following data will be lost:
...
+| :----------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Installation methods           | <p><ul><li>For **kubernetes** we recommend using a dual-region Kubernetes setup with the [Camunda Helm chart](/self-managed/setup/install.md) installed in two Kubernetes clusters.</li><li>For **other platforms**, using alternative installation methods (for example, with docker-compose) is not supported.</li></ul></p>                                                                                         |
```
Comment (maxdanilov) [inline docs/self-managed/concepts/multi-region/dual-region.md:127]: Let's keep it as before (it's not covered, not unsupported)

File: docs/self-managed/operational-guides/multi-region/dual-region-ops.md [hunk 17]
```diff
@@ -139,15 +137,15 @@ desired={<Five viewBox="140 40 680 500" />}
...
+| You have ensured that you fully lost a region and want to start the temporary recovery. <br /> <br /> One of the regions is lost, meaning Zeebe: <br /> - No data has been lost thanks to Zeebe data replication. <br /> - Is unable to process new requests due to losing the quorum <br /> - Stops exporting new data to Elasticsearch in the lost region <br /> - Stops exporting new data to Elasticsearch in the survived region | You should be sure it is lost, and if so, look into measures to prevent it from reconnecting. For example, by utilizing the suggested solution below to isolate your active environment. <br /> <br /> The lost brokers have been removed from the Zeebe cluster. <br /> <br /> Continued processing is enabled, and new brokers in the failback procedure will only join the cluster with our intervention. |
```
Comment (Langleu) [inline docs/self-managed/operational-guides/multi-region/dual-region-ops.md:142]: Please remove this part

> You should be sure it is lost, and if so, look into measures to prevent it from reconnecting. For example, by utilizing the suggested solution below to isolate your active environment.

Not applicable anymore.

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 16 above)
Comment (maxdanilov) [reply docs/self-managed/concepts/multi-region/dual-region.md:127]: yep, exactly, remove "unsupported" in favor of "not documented"

**Review by Langleu — DISMISSED** [review]
thanks for the quick changes 🙇.
Looks good from my side 🚀

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 16 above)
Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md]: I mean in the current docs we explicitly state `not supported`.

Comment (Langleu) [reply docs/self-managed/concepts/multi-region/dual-region.md]: if we want to go to not covered, then the 8.5 docs would need to be adjusted as well since nothing changed between 8.5 and 8.6.

**Review by Langleu — APPROVED** [review]

File: docs/self-managed/concepts/multi-region/dual-region.md (hunk 16 above)
Comment (maxdanilov) [reply docs/self-managed/concepts/multi-region/dual-region.md]: For my memory refreshing, OpenSearch is not compatible with DR, or we did not try because of complexity (as it would not be installed with Helm chart). Basically, checking if there's roadblocks to make DR work with OpenSearch in the future?

Comment (maxdanilov) [reply docs/self-managed/concepts/multi-region/dual-region.md]: We're using 2 separate ES clusters for DR, so that aspect should not be a blocker to have 2 OS clusters to connect to.
